src/screens/ProfileScreen/index.js

```diff
@@ -635,12 +639,12 @@ const ProfileScreen = ({route}) => {
         style={styles.flatlistContainer}
         onScroll={handleScroll}
         ListFooterComponent={isFetchingList ? <ActivityIndicator /> : null}
-        onEndReach={handleOnEndReached}
         initialNumToRender={2}
         maxToRenderPerBatch={2}
         updateCellsBatchingPeriod={10}
         removeClippedSubviews
         windowSize={10}
+        onMomentumScrollEnd={(event) => fetchNextFeeds(event)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onEndReach` prop has been replaced with `onMomentumScrollEnd`. This change could potentially improve performance by loading more items before the user reaches the end of the list. However, ensure that the `fetchNextFeeds` function handles errors and edge cases properly.
```diff
- onEndReach={handleOnEndReached}
+ onMomentumScrollEnd={(event) => fetchNextFeeds(event)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->